msg/RadarPoint.msg

```diff
@@ -0,0 +1,10 @@
+# All variables below are relative to the radar's frame of reference.
+# This message is not meant to be used alone but as part of a stamped or array message.
+
+float32 range                            # Distance (m) from the sensor to the detected point.
+float32 angle                            # Angle (in radians) in the azimuth plane between the sensor and the detected point.
+                                         #   Negative angles are anticlockwise from the sensor.
+float32 elevation                        # Angle (in radians) in the elevation plane between the sensor and the detected point.
+                                         #   Negative angles are below the sensor. For 2D radar, this will be 0.
+float32 doppler                          # The doppler speeds (m/s) of the point.
```

Comment:
Also here for clarity this should be `doppler_velocity`.  Doppler on it's own is not clear, it generally would refer to the physicist whom the Doppler shift was name after. 

Doppler might not be available and it should be documented that a NaN shows unmeasured or unable to measure following the same logic as our more recent patterns developed in MultiEchoLaserScans and NavSatFix and BatteryState.

Reply:
I don't think there's a situation where a radar does not return a doppler, that's the basis of the radar sensor itself. Doppler / velocity are pretty foundational to how a radar operates. I'd be OK with `doppler_velocity` or something similar, but I don't think it would be necessary to document a `NaN` state since that's so foundational.

Reply:
I think range-rate would also be an acceptable term.

Super minor nit, but I would group range/range-rate next to each other in the message definition.

Reply:
The more I think about it, I would actually say that `range_rate` would be correct, because some systems are capable of measuring angle doppler as well, correct?

Reply:
> some systems are capable of measuring angle doppler as well, correct?

It wouldn't surprise me, but I think those are typically rotating or geospatial radars, but another good question for Aaron. I think `range_rate` is an icky name just because its not very clear its a velocity on first glance and I subscribe to the KISS principle. But if we need to distinguish between a range velocity and angle velocity, some distinction may be unavoidable.

Reply:
In terms of radar for object detection 'angle doppler' is not really a thing. Radar detects the distance of objects using an FFT on the signal roundtrip time. Speed based on an FFT of the phase shift across multiple chirps, and angle based on an FFT of the phase shift between receiving antenna. The result of the three FFT's are combined to differentiate between reflections (and thus objects).

I don't like the name range_rate much. To me is not very clear. I'm happy with doppler_velocity however.